header/p2p/options.go

```diff
@@ -112,6 +112,13 @@ func DefaultClientParameters() *ClientParameters {
 	}
 }
 
+// Empty checks if config was initialized.
+func (p *ClientParameters) Empty() bool {
+	return p.MinResponses == 0 && p.MaxRequestSize == 0 &&
+		p.MaxHeadersPerRequest == 0 && p.MaxAwaitingTime == 0 &&
+		p.DefaultScore == 0 && p.MaxPeerTrackerSize == 0
+}
```

Comment:
since bridge nodes are the only nodes that don't run with client, let's just check the type of node and if not bridge, then call validate on client (also make note of that pls in code). Bc this ^ will be annoying to maintain